docker-compose example/docker-compose.example.yml

```diff
@@ -42,6 +42,10 @@ services:
       ## A token can be generated for free here https://account.mapbox.com/access-tokens/
       ## It's a good idea to limit the scope of the token to your own domain, to prevent others from using it.
       MAPBOX_TOKEN: ${MAPBOX_TOKEN}
+      ## Optional: Set the hardware acceleration when encoding videos.
+      ## Support `qsv`, `vaapi`, `nvenc`.
```

Comment:
Instead of this line:
`## If you want to use it, uncomment the option and set the correct value in the .env file